src/Scripting/CoreTestUtilities/ObjectFormatterFixtures/MockDesktopSpinLock.cs

```diff
@@ -15,7 +15,7 @@ namespace ObjectFormatterFixtures
     [DebuggerDisplay("IsHeld = {IsHeld}")]
     internal struct MockDesktopSpinLock
     {
-        private volatile int m_owner;
+        private readonly int m_owner;
```

Comment:
😕 Well this one was interesting

Reply:
@sharwell I didn't expect there would be such bugs. We should file a separate issue for the code fix that made this change right?

Reply:
Having a look at this again, adding readonly and removing volatile seems reasonable to me.
If the field is set only once in the constructor, then I see there is no way to modify it by multiple threads.

Reply:
I would prefer we keep any possible semantic changes out of a code style pr

Reply:
@CyrusNajmabadi Ok. I'll suppress and file a separate issue to look at the suppression later.

```suggestion
#pragma warning disable IDE0044 ' Add readonly modifier - See https://github.com/dotnet/roslyn/issues/47225
        private volatile int m_owner;
#pragma warning restore IDE0044 ' Add readonly modifier
```